en/docs/security-announcements/cve-justifications/2025/GHSA-593m-55hh-j8gv.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "GHSA-593m-55hh-j8gv"
+category: security-announcements
+date: Sep 01 2025
+---
+
+# GHSA-593m-55hh-j8gv
+
+<p class="doc-info">WSO2 Products impacted: No (transitive dependency)</p>  
+<p class="doc-info">Customer actions required: No</p>  
+---
+
+### REPORTED VULNERABILITY
+
+A vulnerability has been identified in **@sentry/browser** where versions prior to **7.119.1** may act as a *prototype pollution gadget*. This means that while the package itself is **not directly exploitable**, it could be abused by an attacker if another prototype pollution vulnerability exists in the application.  
+
+**Affected component:** `@sentry/browser`  
+**Installed version:** 6.19.7  
+**Required version:** 7.119.1  
+
+### REPORTED PRODUCTS
+
+- WSO2 API Manager 4.2.0    
```

Comment:
Do we have to add other product versions as well?

Reply:
The Stoplight version we referenced in the latest apim versions doesn’t include this dependency as a transitive one.